### PR TITLE
Skip Travis for `jupyterlab-github`'s token reset

### PR DESCRIPTION
**Pull request by jakirkham**
xref: https://github.com/conda-forge/jupyterlab-github-feedstock/issues/4
xref: https://github.com/conda-forge/admin-requests/pull/783

### CODE REVIEW

**Comment by jakirkham** [comment]
cc @conda-forge/jupyterlab-github

**Comment by jakirkham** [comment]
Thanks Matt! 🙏

File: token_reset/jupyterlab-github.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # token reset failed for these packages - trying again later
+# --without-travis
```
Comment (h-vetinari) [inline token_reset/jupyterlab-github.txt:2]: I've never seen these magic comments. Do we have them documented somewhere?

**Review by h-vetinari — COMMENTED** [review]
Thanks for helping fix this! 🙏

File: token_reset/jupyterlab-github.txt (hunk 1 above)
Comment (beckermr) [reply token_reset/jupyterlab-github.txt:2]: No that is my bad. We have so little documentation of anything really.

Comment (jakirkham) [reply token_reset/jupyterlab-github.txt:2]: Maybe we can add to [the README section on token resets]( https://github.com/conda-forge/admin-requests#reset-your-feedstock-token )?

**Pull request merged**